crt_portal/cts_forms/templates/forms/complaint_view/show/actions/comment_summary.html

```diff
@@ -1,23 +1,16 @@
-<form
-  id="comment-actions-{{ id_name }}"
-  class="usa-form"
-  method="post"
-  action="{% url 'crt_forms:save-report-comment' report_id=data.pk %}"
-  novalidate
->
```

Comment:
I think my editor must have automatically formatted these lines and some others -- sorry that that makes it harder to review!